core/src/main/python/deploy.py

```diff
@@ -41,12 +41,12 @@
 
 __required_arguments = [
     CommandLineArgUtil.ORACLE_HOME_SWITCH,
-    CommandLineArgUtil.DOMAIN_HOME_SWITCH,
     CommandLineArgUtil.MODEL_FILE_SWITCH
 ]
 
 __optional_arguments = [
     # Used by shell script to locate WLST
+    CommandLineArgUtil.DOMAIN_HOME_SWITCH,
```

Comment:
Is there ever a case where the model for a remote deploy operation might have a domain home token?  If so, what happens if the user did not provide the domain home path on the remote machine?